include/smack/RenameIntrinsics.h

```diff
@@ -0,0 +1,23 @@
+//
+// This file is distributed under the MIT License. See LICENSE for details.
+//
+
+#include "llvm/IR/DataLayout.h"
+#include "llvm/IR/Instructions.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Pass.h"
+#include "llvm/IR/Module.h"
+#include "llvm/IR/User.h"
+#include "llvm/IR/InstIterator.h"
+#include "llvm/IR/Instruction.h"
+
+
+namespace smack {
+
+class RenameIntrinsics : public llvm::ModulePass {
```

Comment:
Add comments explaining what this class does. Please use Doxygen-style comments.

Reply:
Deleted because it is superseded by the IntegerOverlow pass.